database/raft/raft.go

```diff
@@ -510,7 +510,6 @@ func (sv *Service) get(ctx context.Context, key string) ([]byte, error) {
 			err := sv.Set(ctx, "/dummyWrite", []byte(""))
 			if err != nil {
 				// TODO(tessr): should probably return this error
```

Comment:
Aha okay, I'll remove the todo (and add a comment saying it's okay to ignore the error).